.pylintrc

```diff
@@ -62,23 +62,17 @@ confidence=
 # --disable=W".
 disable=attribute-defined-outside-init,
         bad-continuation,
-        bad-whitespace,
         bare-except,
-        blacklisted-name,
         duplicate-code,
         fixme,
-        import-error,
         invalid-name,
-        len-as-condition,
         line-too-long,
         missing-docstring,
-        multiple-statements,
         no-member,
         protected-access,
         R,
+        useless-super-delegation,
```

Comment:
Do not add general exclusions. You can disable checks for individual lines: Add  `# pylint: disable=useless-super-delegation` to the line